.github/workflows/docker.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: |
-            gcr.io/evmchain/taiko-geth
+            us-central1-docker.pkg.dev/evmchain/taiko-geth
```

Comment:
We will need to update the path to be "**_registry/project_id/repo/image_**" will need to be either:
`us-central1-docker.pkg.dev/evmchain/evmchain/taiko-geth
`or
`us-central1-docker.pkg.dev/evmchain/taiko-geth/<some-image-name>`

Reply:
Changed to `us-central1-docker.pkg.dev/evmchain/evmchain/taiko-geth`, lets see https://github.com/taikoxyz/taiko-geth/actions/runs/8756199339